nifi-nar-bundles/nifi-extension-utils/nifi-processor-utils/src/main/java/org/apache/nifi/processor/util/listen/AbstractListenEventProcessor.java

```diff
@@ -199,6 +194,15 @@ public void onScheduled(final ProcessContext context) throws IOException {
         readerThread.start();
     }
 
+    public static InetAddress getNICIPAddress(final String nicIPAddressStr) throws SocketException {
```

Comment:
This is a useful utility method, but if it is going to be `public static`, recommend moving it to a separate utility class.  Also recommend renaming the method to `getInterfaceAddress()`